common/predictive-text/build.sh

```diff
@@ -35,6 +35,14 @@ builder_describe_outputs \
 
 builder_parse "$@"
 
+function do_configure() {
+  verify_npm_setup
+
+  # Configure Web browser-engine testing environments.  As is, this should only
+  # make changes when we update the dependency, even on our CI build agents.
+  playwright install
```

Comment:
Is there a way to make this conditional on having a test action?

Reply:
If we ran it as part of _every_ `test` action, rather than within the project's `configure` action, sure.

Admittedly, once it's been run one time, it generally does complete quite quickly.

Reply:
This is a tricky one. There may be environments where we can't run browsers, even headlessly, due to lower resources for example. But I think conceptually the step belongs in `configure`, because it is about configuring the dev environment. 

In the general case, we probably want a configured environment to be able to run the build and the tests.